src/styles/card_styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
Consider combining the `background` properties into a shorthand to make the code cleaner and easier to read. For example: `background: url(../images/star.svg) no-repeat center;`.